editor/utils/url.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { parse, format } from 'url';
+
+/**
+ * Appends arguments to the query string of the url
+ *
+ * @param  {String} url   URL
+ * @param  {Object} args  Query Args
+ *
+ * @return {String}       Updated URL
+ */
+export function addQueryArgs( url, args ) {
+	const parsedUrl = parse( url, true );
+	const query = { ...parsedUrl.query, ...args };
```

Comment:
> it could be marginally more performant to mutate `parsedUrl` throughout this function

IMO, this is only important if we expect the function in question to be called **extremely** frequently.  Otherwise I think the code clarity from using the spread operator is worth it.